go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/iancoleman/strcase v0.3.0
 	github.com/martinlindhe/base36 v1.1.1
 	github.com/open-policy-agent/opa v0.57.0
-	github.com/pomerium/pomerium v0.23.1-0.20231106215759-ffca3b36a9da
+	github.com/pomerium/pomerium v0.23.1-0.20231109152321-34187e8ba551
```

Comment:
It looks like 0-24-0 is now at pomerium/pomerium@34187e8. Do we want to match that here as well?